packages/theme/src/types/IFabricConfig.ts

```diff
@@ -18,4 +18,9 @@ export interface IFabricConfig {
    * The mergeStyles stylesheet config.
    */
   mergeStyles?: IStyleSheetConfig;
+
+  /**
+   * Serialized form of Stylesheet used for rehydration.
+   */
+  serializedStylesheet?: ISerializedStylesheet;
```

Comment:
I am not happy about this type definition here, It leads to duplication of definition for `_global` in Stylesheet (https://github.com/microsoft/fluentui/pull/19888/files#diff-6dc4b70fa61de4b9c9dcf834e4f50bcf8b9df694264ee69d92edf9b5aea82dceR99)